Create setup script for Azure Key Vault (#1909)
seattle-uat/civiform: cloud/deploys/bin/azure/README.md

# azure/bin

> This directory includes a scripts to help with setting up the staging or production environment

Reviewer: nit:
```suggestion
This directory includes scripts to help with managing staging and production environments in Azure.
```

After running this you should be able to run postgres commands directly. 

### Known issues
To run a pg_dump/restore on a newly provisioned machine will need to 
update the postgres version (instructions at the stack overflow below)
if it's a newly provisioned machine follow the stackoverlow linked [here](https://stackoverflow.com/questions/52765873/how-to-upgrade-to-postgresql-11-for-ubuntu-18-04)

## Key Vault

## Key Vault Setup

> Creates a key vault instance and stores generated values for the postgres admin password and application secret key. Before running this script, make sure that you have the "Key Vault Administrator" permission assigned to your Azure user account. For instructions on how to assign Azure roles, see [Assigning Azure Roles using Azure CLI](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-cli).

Reviewer 2: I had to add `Key Vault Secrets Officer` but not sure if the administrator is superset of that

Author: I think Key Vault Administrator gives more permission, because it allows you to store secrets, certificates, and public/private keys. Secrets officer would probably be better for now.
